github_client/github.go

```diff
@@ -1,11 +1,12 @@
-package github
+package github_client
```

Comment:
Potential Bug: The dependency "golang.org/x/crypto" is pointing to an older version. Please ensure that there's no security threat if this version is used.